testflight/suite_test.go

```diff
@@ -104,6 +105,11 @@ var _ = SynchronizedBeforeSuite(func() []byte {
 		config.ATCGuestPassword = atcGuestPassword
 	}
 
+	caCert :=  os.Getenv("CA_CERT")
```

Comment:
If we move forward with putting mutual tls as an override file then we don't need to update testflight to handle mutual tls, which I would prefer. If we want to test mutual tls we should add a topgun test. There are some changes happening to topgun right now so I wouldn't bother adding test coverage for that **for now.**

Reply:
removed